api/client/client.go

```diff
@@ -2434,10 +2434,11 @@ func (c *Client) DeleteAllNodes(ctx context.Context, namespace string) error {
 }
 
 // StreamSessionEvents streams audit events from a given session recording.
-func (c *Client) StreamSessionEvents(ctx context.Context, sessionID string, startIndex int64) (chan events.AuditEvent, chan error) {
+func (c *Client) StreamSessionEvents(ctx context.Context, sessionID string, startIndex int64, format string) (chan events.AuditEvent, chan error) {
```

Comment:
What do you think about putting the format inside the context instead of an explicit parameter?

I don't love making format a required argument because the stream session events API doesn't know anything about formats - that all happens downstream. Adding a required argument for something that's only used for some event metadata feels wrong.

Reply:
Yeah, I agree with this (especially after seeing so many places requiring this parameter, which would be empty by default). I'll update it and make another PR (there will be fewer places to change).